orthopairs/README.md

```diff
@@ -0,0 +1,54 @@
+<h2> Orthopairs: Protein ortholog file generation</h2>
+
+To see how to run this script, check out the <a href="https://github.com/reactome/data-release-pipeline/new/feature/orthopairs/orthopairs#-running-orthopairs-">Running Orthopairs</a> section.
```

Comment:
Nitpick: Java applications are not scripts.

Reply:
program?